tests/cilium_http_integration.cc

```diff
@@ -7,6 +7,7 @@
 #include <memory>
 #include <string>
 
+#include "envoy/http/codec.h"
```

Comment:
this include is marked as unused in my IDE :face_with_diagonal_mouth:

Reply:
I think clang-tidy asked me to add these. If we add the clang tidy CI check, then we'll have to keep these.

We are using `Http::CodecClient::Type::HTTP1`, which is defined in `source/common/http/codec_client.h` as:
```
class CodecClient ... {
  using Type = Envoy::Http::CodecType;
```

So we are "using" the definition in `envoy/http/codec.h`.

Reply:
maybe we need a `// IWYU pragma: keep` here?

Reply:
Added pragma